internal/rego/rego.go

```diff
@@ -177,16 +180,36 @@ func (r Rego) PolicyID() string {
 // found in the header comment of the rego file.
 func (r Rego) Description() string {
 	var description string
+	var handlingCodeBlock bool
 	for _, comment := range r.headerComments {
-		if strings.HasPrefix(comment, "@") {
+		if strings.HasPrefix(strings.TrimSpace(comment), "@") {
```

Comment:
Should use `commentHasKeyword()` here for consistency

Reply:
yeah, I hesitated adding it there because I didn't like using `@` (and the code blocks below) as a "keyword". Should just rename the method.